pulp_ansible/app/tasks/utils.py

```diff
@@ -13,8 +13,13 @@ def get_page_url(url, page=1):
     """Get URL page."""
     parsed_url = urlparse(url)
     new_query = parse_qs(parsed_url.query)
-    new_query["page"] = page
-    new_query["page_size"] = PAGE_SIZE
+
+    if "v1" in url or "v2" in url:
```

Comment:
Also can the check include `/api/v2` and `/api/v3` instead? this will avoid complications when collection names have v2 in them for example

Reply:
it would not work with `/api/automation-hub/v3/collections/`